js/indicia.datacomponents/jquery.idc.esDownload.js

```diff
@@ -296,6 +296,11 @@
       $.extend(currentRequestData, columnSettings);
       // Reset.
       rowsToDownload = null;
+      // If there is an associated download template select control,
```

Comment:
Minor comment - just suffixing -select to the control ID results in an ID which doesn't describe that it's a template selector. Maybe -template would be a better suffix? Also less likely to clash with other controls if other features are added to the download.

Reply:
I will change.